helpers/api.js

```diff
@@ -0,0 +1,321 @@
+const data = [
+  {
+    Id: "1",
+    Solution: "Einkauf",
+    Team: "AA",
+    BPE: "Sebastian Brucker",
+    BSEINT: "Jochen Maier",
+    BSEGR: "Helmut Schön",
+    LeadDevloper: "Veit Maurer",
+    CBO: "Ahmet Peros",
+    SupportGroup: "supp-einkauf",
+    Modules: [
+      {
+        Modul: "ABC",
+        Developers: [
+          {
+            DevloperId: "1",
+            Firstname: "Thomas",
+            Lastanme: "Müller",
+          },
```

Comment:
I recommend you to write the keys in `camelCase`. It's a naming convention that is used by most APIs etc. Also for later use in your code it would make things look better and it's less confusing (example: `<li key={data.id}>{data.solution}</li>` instead of `<li key={data.Id}>{data.Solution}</li>`

```suggestion
    id: "1",
    solution: "Einkauf",
    team: "AA",
    bpe: "Sebastian Brucker",
    bseint: "Jochen Maier",
    bsegr: "Helmut Schön",
    leadDevloper: "Veit Maurer",
    cbo: "Ahmet Peros",
    supportGroup: "supp-einkauf",
    modules: [
      {
        modul: "ABC",
        developers: [
          {
            devloperId: "1",
            firstName: "Thomas",
            lastName: "Müller",
          },
```

Reply:
Thank you. I will updagte this in the next feature branch

Reply:
Normally you should update it in this PR if you follow the Git Workflow, so don't forget about it 😉

Reply:
i agree with marc. maybe create a fix branch in the meantime and post a new PR for that branch, before continuing on the feature branch. that way we can minimize divergence.